app/views/state_file/questions/nj_veterans_exemption/edit.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+    title = t(".title")
+    content_for :page_title, title
+%>
+<% content_for :card do %>
+  <h1 class="h2"><%= t(".title") %></h1>
```

Comment:
[dust] you could reuse the local variable `title` allocated on line 2. i don't have a strong preference for either setting local variables or calling `t` multiple times, but would prefer consistency.